assets/metrics/bag-of-words_metrics.json

```diff
@@ -0,0 +1 @@
+{"Accuracy": 0.36406666666666665, "F1 score": 0.656114532318378, "Average precision score": 0.3558413160535066}
```

Comment:
Is this part needed for the WandB part?

Reply:
Seemed like an important part of the DVC pipeline, at least that's how it seemed from the lectures/tutorials. It could also help with wandb.